src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -145,6 +147,23 @@ function useSidebarMetadataFetcher(
         [api, onApiError, file, deleteMetadataInstanceSuccessCallback],
     );
 
+    const updateMetadataInstance = () => {
+        // to be implemented in the next ticket
+    };
+
+    const handleCreateMetadataInstance = React.useCallback(
+        (template: MetadataTemplateInstance, successCallback): void => {
```

Comment:
We already differentiate template instances and templates. I think, the name should reflect the data type

Reply:
changed the name